### PR TITLE
added weather feature by full webscrapping only

### PR DESCRIPTION
**Pull request by Vincexodus**
Getting weather information from websites :)

### CODE REVIEW

File: Assets/Scripts/UI/BGCanvas.cs [hunk 1]
```diff
@@ -41,9 +40,9 @@ public partial class BGCanvas : MonoBehaviour
   public TextMeshProUGUI performance;
   [Header("Weather")]
   public TextMeshProUGUI temperature;
-  public TextMeshProUGUI windspeed;
+  public TextMeshProUGUI windspeed, windDirection;
   public TextMeshProUGUI description;
-  public TextMeshProUGUI pressure;
+  public TextMeshProUGUI humidity;
   public TextMeshProUGUI Location;
```
Comment (nixon-voxell) [inline Assets/Scripts/UI/BGCanvas.cs:46]: try to use camelCase here Location -> location

Comment (Vincexodus) [reply Assets/Scripts/UI/BGCanvas.cs:46]: yikes didnt notice that sry

File: Assets/Scripts/UI/BGCanvas.cs [hunk 2]
```diff
@@ -41,9 +40,9 @@ public partial class BGCanvas : MonoBehaviour
   public TextMeshProUGUI performance;
   [Header("Weather")]
   public TextMeshProUGUI temperature;
-  public TextMeshProUGUI windspeed;
+  public TextMeshProUGUI windspeed, windDirection;
```
Comment (nixon-voxell) [inline Assets/Scripts/UI/BGCanvas.cs:43]: here also windSpeed

File: Assets/Scripts/UI/BGCanvas.cs (hunk 1 above)
Comment (nixon-voxell) [reply Assets/Scripts/UI/BGCanvas.cs:46]: np, after change ler just commit and push again, this push request should automatically update.

**Pull request merged**